app/src/main/java/com/kickstarter/models/QualtricsIntercept.kt

```diff
@@ -9,4 +9,8 @@ enum class QualtricsIntercept(private val prodId: String, private val testId: St
             else -> testId
         }
     }
+
+    fun impressionCountKey(packageName: String): String {
```

Comment:
I was under the _impression_ (lol) that we didn't need the impression count anymore if we're confident that Qualtrics won't prompt us once we've taken the survey? Anyway, suppose no real harm in keeping it unless you wanted to simplify your code without it.

Reply:
This count is so we don't show users the prompt more than 3 times. Otherwise, we'd be hounding them until they took the survey.